makejdk-any-platform.1

```diff
@@ -106,6 +106,9 @@ It is known issue: https://github.com/adoptium/temurin-build/issues/3862
 .BR \-\-debug-docker
 debug OpenJDK build script in a docker container. Only valid if \fB-D\fR is selected.
 .TP
+.BR \-\-disable-adopt-branch-safety
+disable the default check for tags/branches in case of \fBtemurin\fR as --build-variant.
```

Comment:
What's the check?  More context will help

Reply:
Tbh I'm not sute. i think itis checking whether you are in correct repo, in correcy branch with correct tags.I will try to diga bot more

Reply:
Yes and the comment that it is checking if you are in the correct tag is good context to add.

Reply:
@karianna  ok. In that case I have to verify that I understood it correctly. Tahnk you!

Reply:
The more precise description is actually not related to "tags" but the detection of the TEMURIN_MARKER_FILE, which is the file README.JAVASE, which is only present in the Adoptium "mirror" : https://github.com/adoptium/jdk21u/blob/dev/README.JAVASE. This check is to ensure Temurin is building from the Adoptium mirror source, and not something else..., hence the use of the word "safety".
The "fake tag" bit is just an additional bit to avoid the missing _adopt tag associated with not using an Adoptium mirror.

Reply:
Thanx a lo, updated.